portfolio/src/main/webapp/style.css

```diff
@@ -28,6 +28,16 @@ img {
     background-color: #008080;
 }
 
+.card {
+    width: 50%;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
optional: put these in alphabetical order

This is what we do at Google, it's a little weird because some styles work best together so putting them one after the other makes sense, but alphabetical helps keep the code clean and makes it easier to search for an attribute.

You don't have to update this for this project, but I think I'll try to enforce it for the group project